config/projects.yml

```diff
@@ -20,3 +20,78 @@ parameters:
         - coding-standard
         - doctrine1
         - instantiator
+
+    doctrine.website.projects_configuration:
+        cache:
+            integrations:
+                -
+                    type: Symfony
+                    name: DoctrineCacheBundle
+                    url: https://github.com/doctrine/DoctrineCacheBundle
+
+        dbal:
+            integrations:
+                -
+                    type: Symfony
+                    name: DoctrineBundle
+                    url: https://github.com/doctrine/DoctrineBundle
+
+                -
+                    type: Zend Framework
+                    name: DoctrineModule
+                    url: https://github.com/doctrine/DoctrineModule
+
+
+        doctrine2:
+            integrations:
+                -
+                    type: Symfony
+                    name: DoctrineBundle
+                    url: https://github.com/doctrine/DoctrineBundle
+
+                -
+                    type: Laravel
+                    name: Laravel Doctrine ORM
+                    url: https://github.com/laravel-doctrine/orm
```

Comment:
This is not official (not under Doctrine org) extension. Do we really want to list these? This way we may end up with many PRs to add unofficial extensions for any framework/tool out there.